File: WinClient/Web/NetInteraction/AuthResponse.cs
using Context;
using Newtonsoft.Json;

namespace Web.NetInteraction
{
    public class AuthResponse
    {
        [JsonConverter(typeof(AuthResultConverter))]
        public AuthResult? Result { get; set; }
        public UserData? Data { get; set; }
        public string? PermCts { get; set; }
    }
}

File: Server/src/net/requests/auth/AuthResponse.java
package net.requests.auth;

import entities.UserAccount;

public class AuthResponse {
    public AuthResult Result;
    public UserAccount Data;
    public String PermCts;
    public AuthResponse result(AuthResult result) {var a = this; a.Result = result; return a;}
    public AuthResponse data(UserAccount data) {var a = this; a.Data = data; return a;}
    public AuthResponse permCts(String PermCts) {var a = this; a.PermCts = PermCts; return a;}
}
